apps/web/src/components/layout/components/SideNav.tsx

```diff
@@ -121,6 +122,10 @@ export function SideNav({}: Props) {
     },
   ];
 
+  if (!isMultiTenancyEnabled) {
```

Comment:
You can use condition property of the menu item - like in Changes. 

```
{
 icon:<Building />,
.....,
condition: isMultiTenancyEnabled
```

Reply:
i was not aware of the condition param 😅
much better, thanks!